scripts/render_platform_manifests.py

```
@@ -0,0 +1,365 @@
+        _parse_bool(str(enable_cert_manager_raw)),
+        _parse_bool(str(enable_external_dns_raw)),
+        _parse_bool(str(enable_vault_eso_raw)),
+        _parse_bool(str(enable_cnpg_raw)),
+    )
+
+
+def _resolve_paths() -> tuple[Path, Path, Path]:
+    """Resolve paths: runner_temp, output_dir, github_env."""
+    runner_temp_raw = resolve_input(
+        None,
+        InputResolution(env_key="RUNNER_TEMP", default=Path("/tmp"), as_path=True),
```

Ruff (0.14.13)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)